rpc/filters.go

```diff
@@ -13,13 +14,24 @@ import (
 	Used to set the criteria passed in from RPC params
 */
 
-// Filter can be used to retrieve and filter logs.
+type filterType = string
```

Comment:
q: why not have it just as a string if it's private?

Reply:
it's nice to have `filter.typ` as a specific type, but string is fine